src/driver.ts

```diff
@@ -147,10 +175,27 @@ export const setTemperatureInKelvin = (
   );
 };
 
+/**
+ * Gets the temperature of your Logitech Litra device
+ *
+ * @param {Device} device The device to get the temperature for
+ * @returns {number} The current temperature in Kelvin
+ */
+export const getTemperatureInKelvin = (device: Device): number => {
```

Comment:
What do these read functions return if the device is off?

Reply:
They return the current value regardless if the light is powered on/off. Quite convenient.